src/js/function/enterInputTryNumber.js

```diff
@@ -0,0 +1,7 @@
+import { confirmTryNumber } from "./confirmTryNumber.js";
+
+export const enterInputTryNumber = (event) => {
+  if (event.keyCode === 13) {
+    confirmTryNumber();
+  }
+};
```

Comment:
```suggestion
export const enterInputTryNumber = filterKeyEnter(() => {
  confirmTryNumber();
});
```
이렇게 표현할 수 있게 `filterKeyEnter` 라는 함수를 하나 만들어주면 어떨까요?
다른 함수에서도 사용되고 있는 것 같아서요!